tests/cypress/integration/frontend/split-content.spec.js

```diff
@@ -0,0 +1,82 @@
+describe('Block visibility tests for non WM-enabled users', () => {
```

Comment:
Does it mess things up to change wording to something like "Check visibility of content blocks". Just to avoid anyone thinking that this is blocking visibility.